Source/WebCore/dom/Element.h

```diff
@@ -523,10 +523,7 @@ class Element : public ContainerNode {
 
     virtual bool childShouldCreateRenderer(const Node&) const;
 
-    bool hasPendingResources() const { return hasNodeFlag(NodeFlag::HasPendingResources); }
-    void setHasPendingResources() { setNodeFlag(NodeFlag::HasPendingResources); }
-    void clearHasPendingResources() { clearNodeFlag(NodeFlag::HasPendingResources); }
-    virtual void buildPendingResource() { };
+    virtual void buildPendingResource() { }
```

Comment:
Do we need it here? Non-SVG elements override it???

Reply:
No, I don't think so. I'll make this change at the same time.